src/services/ui/src/features/package/index.tsx

```diff
@@ -38,8 +38,8 @@ export const DetailsContent: FC<{ id: string }> = ({ id }) => {
   const title =
     (() => {
       switch (data._source.authority) {
-        case Authority["1915b"]:
-        case Authority["1915c"]:
+        case AUTHORITY["1915(b)"]:
+        case AUTHORITY["1915(c)"]:
```

Comment:
If `_source.authority` is typed out as `Authority`, we can replace this constant with strings. It wasn't possible to do that with enums:

```ts
switch (data._source.authority) {
  case "1915(b)":
  case "1915(c)":
```